integration/src/test/scala/org/knora/webapi/messages/util/search/gravsearch/prequery/GravsearchToCountPrequeryTransformerSpec.scala

```diff
@@ -201,7 +199,7 @@ class GravsearchToCountPrequeryTransformerSpec extends CoreSpec {
   val transformedQueryWithDecimalOptionalSortCriterionAndFilterComplex: SelectQuery =
     SelectQuery(
       fromClause = None,
-      variables = Vector(
+      variables = List(
```

Comment:
Isn't `Vector` faster than `List`? Did you check if only changing `ArrayBuffer` to `Vector` above improves the results?

Reply:
> Isn't Vector faster than List? 

Not in all cases, and does it really matter in the tests. `Vector` is a more efficient data structure for random access and updates, while a `List` is more efficient for sequential access and prepending elements.

I had started using `List` because that was the `Seq` actually returned. However, the scalatest matchers do not mind the type of the collection, only the contents.